app/actions/websocket/teams.ts

```diff
@@ -49,3 +52,90 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: any) {
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const teamFromData = JSON.parse(msg.data.team);
+    const team: Team = {
+        ...teamFromData,
+    };
+
+    database.operator.handleTeam({
+        teams: [team],
+        prepareRecordsOnly: false,
+    });
+}
+
+export async function handleTeamAddedEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const user = await queryCurrentUser(database.database);
+    if (!user) {
+        return;
+    }
+
+    const [team, memberships] = await Promise.all([
+        fetchMyTeam(serverUrl, msg.data.team_id),
+    ]);
+
+    console.log('team', team);
+    console.log('memberships', memberships);
```

Comment:
You need this?

Reply:
I'm quite confused at this point. fetchMyTeam performs a call to teh server, but also updates the state. Calling this function will add the new team to the Team sidebar.

Should use this fucntion, or handle the actions from V1 individually?

I'm ok with working to modify the websocket data from the server, if you prefer that route.